resources/com.sap.piper/pipeline/stageDefaults.yml

```diff
@@ -18,3 +18,7 @@ stages:
   Compliance: {}
   Promote: {}
   Release: {}
+  postAction:
```

Comment:
why camelCase here instead of starting with uppercase as for other stages?

Reply:
changed to `Post Actions`